### ISSUE TITLE
bug (YouTube): Cannot expand video descriptions on tablet

### ISSUE THREAD

**Issue by j1elo** [ReVanced/revanced-patches#2538]
### Bug description

On a tablet, trying to expand the video description does nothing when tapping on the title or on the `...more` link below a playing video.

However, doing exactly the same in parallel with a phone, it works fine and the video description expands and shows up.

Steps to reproduce:

* Open patched YouTube.
* Tap on any video.
* Tap on the video's title, or on the whole area of the title, where the global view count is, or where the `...more` indicator is.

More details:

* Tested with a clean install (no settings changed from defaults) on both tablet and phone.
* The network and the devices don't have any kind of ad-blocker such as AdGuard.

Thanks a lot for this app and for all the effort it takes to make it... I only hope to help make it a tiny bit better.

Merry Christmas!

### Error logs

```shell
$ logcat | grep -E 'AndroidRuntime|YouTube|Gms'
```

Every time the description is attempted to expand:

```
12-24 17:06:37.287  2765  2765 E YouTube : EngagementPanelController: failed to get a valid EngagementPanel instance.
```

### Additional context

Version info:

* YouTube v18.45.43, patched on a PC with `revanced-cli`.
* Vanced MicroG v0.2.24.220220
* revanced-cli v4.3.0
* revanced-integrations v1.0.0
* revanced-patches v3.1.0

Command used for patching:

```sh
java -jar revanced-cli-*.jar patch \
    --patch-bundle revanced-patches-*.jar \
    --merge revanced-integrations-*.apk \
    --purge \
    com.google.android.youtube_*.apk
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
Follow this and report back:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/ba2fb861-155d-49da-98ce-f0c6525dda71)

**Comment by j1elo** [comment]
FIXED: This issue doesn't happen if "Enable tablet layout" is enabled.

So for anyone that maybe happens to have the same issue and finds this:

you need to go to Settings > ReVanced > Layout, and turn "Enable tablet layout" ON.

I'll close this one as it really is just caused for the same reason as https://github.com/ReVanced/revanced-patches/issues/2537
